client/service_inline.go

```diff
@@ -65,7 +65,7 @@ type serviceInlineClient struct {
 type ServerInitialInfo interface {
 	Endpoints() endpoint.Endpoint
 	Option() *internal_server.Options
-	GetServiceInfo() *serviceinfo.ServiceInfo
+	GetServiceInfos() map[string]*serviceinfo.ServiceInfo
```

Comment:
cc @Duslia

Reply:
fyi: @Duslia tested and found it was fine :)